### DISCUSSION TITLE
Summarization and speeding up responses

### DISCUSSION THREAD

**Discussion by femi-orokunle** [deepgram/community#381] [General help] [answered]
Note: this was copy/pasted from Discord; I'm not sure which community is more active for getting help so apologies for duplicating my question.

Anyone have any guidance they can share re: summarization and getting faster API responses? 

I have a few questions related to my use-case: 

1) Does requesting summarizations from Nova 2 alongside the transcription greatly increase response times? I'm uploading 10-15 minute audio files and getting my transcription + summary can take 2-5 minutes which is longer than I'd expect (maybe I'm wrong to expect faster?

2) Is there a way to run the two requests in parallel in separate instances, to make it faster? E.g. one request for the transcript, one for the summary 

3) What other optimizations can I look at to get data back ASAP? Is using URLs always going to be inherently slower than other methods?

**Comment by DamienDeepgram** [comment]
Can you share a request id for an example request where this slowness is happening so we can take a look into it for you?

**Reply by femi-orokunle** [reply]
Here's the ID: 

6bce1a99-3b62-42e9-a23f-b052e81625c6

**Reply by DamienDeepgram** [reply]
I checked on our side and this request took 15 seconds according to our logs.

Is it possible there were some network issues on your side when this request took place?

If you run the same request do you see the same result?

**Reply by DamienDeepgram** [reply, marked as answer]
I tested with a sample 5 min audio file and it took 10 seconds for me just now
